spec/rails_helper.rb

```diff
@@ -7,6 +7,23 @@
 require 'rspec/rails'
 require_relative 'support/controller_macros'
 
+OmniAuth.config.test_mode = true
+omniauth_hash = {'provider' => 'github',
+                 'uid' => '12345',
+                 'info' => {
+                   'name' => 'test',
+                   'email' => 'test@github.com',
+                   'nickname' => 'test'
+                 },
+                 'extra' => {
+                   'raw_info' => {
+                     'location' => 'San Francisco',
+                     'gravatar_id' => '123456789'
+                   }
+                 }}
```

Comment:
@sinpai can't we use such style?
```ruby
omniauth_hash = {
  'provider' => 'github',
  'uid' => '12345',
  'info' => {
    'name' => 'test',
    'email' => 'test@github.com',
    'nickname' => 'test'
  },
  'extra' => {
    'raw_info' => {
      'location' => 'San Francisco',
      'gravatar_id' => '123456789'
    }
  }
}
```
Imagine, that you have to rename `omniauth_hash`, it will be a pain to add/remove outline of a hash